src/random_test.cpp

```diff
@@ -3041,7 +3041,9 @@ void Thd1::run_some_query() {
     ddl_query = true;
     Table *table = Table::table_id(Table::TEMPORARY, i, this);
     if (!execute_sql(table->definition(), this))
-      throw std::runtime_error("Create table failed " + table->name_);
+      thread_log << thread_id << " Error create table failed " << table->name_
```

Comment:
Why for normal tables we are throwing runtime exception (pasted below) and just logging error for temporary tables?
`if (!execute_sql(table->definition(), this))
        throw std::runtime_error("Create table failed " + table->name_);`
 
 What will happen if server crashes during normal table creation after metadata is loaded from step file ?

Reply:
Normal table creation does not happen after metadata is loaded from step file. It happens in  first step  or if --prepare is pass. Currently we consider it as fatal error. if table is failed to create pstress can't create the metadata file.

Reply:
If user has passed --recreate-table option, then normal tables will be created even after metadata is loaded. So, in that case, will pstress exit creating the step file ?

Reply:
--recreate-table is just drop and create. We don't crash incase creates fails.